src/applications/appeals/shared/utils/focus.js

```diff
@@ -28,7 +28,7 @@ export const focusIssue = (_index, root, value) => {
         focusElement('.edit-issue-link', null, card);
       }
     } else {
-      scrollTo('h3');
```

Comment:
Noticed `scrollToTop` was the most recent update for focusing. Can you confirm if that's the preferred focus method we want to use to share?